.goreleaser.yml

```diff
@@ -12,6 +12,9 @@ builds:
     - 386
     - arm64
     - mips64le
+  ignore:
```

Comment:
QQ: why do we need this?

Reply:
```
• DEPRECATED: skipped darwin/arm64 build on Go < 1.16 for compatibility, check https://goreleaser.com/deprecations/#builds-for-darwinarm64 for more info.
```

Reply:
We don't need this as we're building with Go v1.16 in the upcoming action.